Cartfile

```diff
@@ -1,4 +1,4 @@
-github "attaswift/BigInt" ~> 3.1
-github "krzyzanowskim/CryptoSwift" ~> 0.8
+github "crossroadlabs/BigInt" "master"
```

Comment:
Don't change the dependencies please. I am maintainer of all those libraries and you can always submit PRs there if you think something's missing.

Reply:
Oh sorry, I guess I missed you already changed it back...

Reply:
Yep. BTW, I will submit PR then. BigInt can't be built with Carthage on case-sensitive FS. A wrong case of some folders inside Xcode project settings

Reply:
Sure. Just tag me over there and I will merge it ASAP.